### PR TITLE
fixed: echo the footer in the recipient language

### PR DESCRIPTION
**Pull request by oseg**

### CODE REVIEW

File: views/default/html_email_handler/notification/body.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	$notification_footer = elgg_echo('html_email_handler:notification:footer:settings', [
 		"<a href='{$settings_url}'>",
 		'</a>',
-	]);
+	], $recipient->language);
```
Comment (jeabakker) [inline views/default/html_email_handler/notification/body.php]: I believe you can use `$language` which is defined at the start of the file and is the same language as available in the notification hooks (which is the recipient language)

Comment (oseg) [reply views/default/html_email_handler/notification/body.php]: In most of the cases, the recipient user is the logged in user. But, in some cases as #43, the recipient user language is not the same language as the logged in user language.

Comment (jeabakker) [reply views/default/html_email_handler/notification/body.php]: In most cases the **sender** is the logged in user, the recipient is somebody else.

I checked i'm not passing language yet. And i think `$language` should instead be defaulted to `$recipient->getLanguage()`

Comment (oseg) [reply views/default/html_email_handler/notification/body.php]: I agree

**Comment by jdalsem** [comment]
if you update the PR i can merge it

**Comment by jeabakker** [comment]
thank you

**Comment by oseg** [comment]
you're welcome

**Pull request merged**